pyomo/common/tests/test_fileutils.py

```diff
@@ -72,6 +73,11 @@ def test_this_file(self):
             'print(this_file())'
         )[1].strip(), '<stdin>')
 
+    def test_this_file_dir(self):
+        print(_this_file_dir)
```

Comment:
Is this a debugging line that should be removed?

Reply:
Yes. fixed.